### ISSUE TITLE
Epic2 with no input

### ISSUE THREAD

**Issue by nservant** [biocore-ntnu/epic2#44]
Hi,
I tried to run Epic2 without control/input data.  
It's running, but the output file is different and I would like to valide with you that it's ok because I did not find any information on that on the doc.

When running Epic2 with input control I have ;
```
#Chromosome	Start	End	PValue	Score	Strand	ChIPCount	InputCount	FDR	log2FoldChange
```

Now wihtout control I have :
```
#Chromosome	Start	End	ChIPCount	Score	Strand
```

In practice, it would be great if the output format could be the same, even if some column are set to NA for instance.
Many thanks

**Comment by endrebak** [comment]
I guess I am missing docs for the case where there is no background :/

Why do you want to include always NA columns in the files? Perhaps we can find a workaround :)

**Comment by nservant** [comment]
This is just because I included Epic2 in a nextflow pipeline that we are using to analysis any kind of data (with or without replicates). Therefore, it is easier to manage one type of output file regardless the options you are using.
But that's fine, I just update a bit my code and it's ok

**Comment by millerh1** [comment]
I thought I would drop this here, since the conversation seemed relevant... But I think that EPIC2 may have a very high false positive rate without an input control. Here is a screencap from IGV showing the peaks (18,976) found on an input control by EPIC2. The input control has almost 0 peaks when using MACS2 btw. I was wondering, do you have any suggestions for how to go about setting parameters to control the false positive rate without the input control?

Here is an example, all of Chr3 in hg38. I've included the S9.6 sample (this is DRIP-Seq, a kind of diffuse domain sequencing) -- and the corresponding Input control. In both the S9.6 and Input samples, I did not use a control -- I put both in as the treatment in separate EPIC2 runs. 

```shell
epic2 -t "bam/MSC_input_S15/MSC_input_S15.hg38.bam" -gn hg38 -o "peaks/MSC_Input.epicpeaks.bed"
epic2 -t "bam/MSC_input_S15/MSC_input_S15.hg38.bam" -gn hg38 -o "peaks/MSC_Input.epicpeaks.bed"
```

Also the empirical FDR for the input control was reported as ~0.053 -- the empirical FDR for the S9.6 was report around ~0.021

![image](https://user-images.githubusercontent.com/44813811/121443406-0ea38c80-c953-11eb-92bf-ac7aec9e226c.png)

**Comment by millerh1** [comment]
With e value set to 100, I saw 1/2x the number of peaks and an empirical FDR of ~0.01

I then tried dropping the e value to 10, which led to 1/4x the number of peaks and an empirical FDR of ~0.002

I then tried e value of 1 (lowest possible), which led to 1/8 the number of peaks an empirical FDR of ~0.0004

It seems like dropping the evalue didn't have as much effect on the treatment sample (S9.6). Anyways, here's what these all look like in a region of Chr19 (below). Hope this helps!

![image](https://user-images.githubusercontent.com/44813811/121445883-f4b87880-c957-11eb-984f-2043a6442248.png)

**Comment by endrebak** [comment]
Yes! ChIP-seq data is 95% noise so input should always be used :)

**Comment by millerh1** [comment]
Right @endrebak I think the issue was that epic2 was calling so many peaks from input samples which are, by definition, not IP'ed

**Comment by endrebak** [comment]
I do not have any suggestions really. I think epic2 should be used with
input. I just added the option to not use input just to be
feature-equivalent with SICER.

**Comment by millerh1** [comment]
That makes sense! @endrebak Maybe it would be worthwhile to include a disclaimer about using epic2 without an input control, seeing that it can call so many peaks from pure noise. It took me a lot of experimenting to realize this limitation
